hippiestation/code/game/gamemodes/vampire/vampire_bite.dm

```
@@ -0,0 +1,20 @@
+/mob/living/carbon/human/proc/handle_vamp_biting(var/mob/living/carbon/human/M)
+	if(!is_vampire(M) || M == src || M.zone_selected != "head")
```

so it silently fails if the target is another vampire?